src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString
+    .split(';')
+    .filter((style) => style.trim())
+    .map((style) => {
+      const [property, value] = style.split(':');
+
+      return [property.trim(), value.trim()];
+    })
+    .reduce((acc, [property, value]) => {
+      acc[property] = value;
+
+      return acc;
+    }, {});
+
+  return stylesArray;
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`, `object` `arr`, `obj`, `result`. In this case, `stylesArray` is misleading because the final result is not an array but an object. Consider renaming it to better reflect its content, such as `stylesObject`.